CITATION.cff

```diff
@@ -0,0 +1,66 @@
+cff-version: "1.2.0"
+authors:
+- family-names: Ittah
+  given-names: David
+  orcid: "https://orcid.org/0000-0003-0975-6448"
+- family-names: Asadi
+  given-names: Ali
+- family-names: Lopez
+  given-names: Erick Ochoa
```

Comment:
```suggestion
- family-names: Ochoa Lopez
  given-names: Erick
```

Reply:
Thanks @erick-xanadu :) This file was automatically generated by JOSS and I didn't proofread it carefully!